contracts/src/v0.8/automation/upkeeps/LinkAvailableBalanceMonitor.sol

```diff
@@ -95,7 +106,7 @@ contract LinkAvailableBalanceMonitor is ConfirmedOwner, Pausable, AutomationComp
     address[] calldata addresses,
     uint96[] calldata minBalances,
     uint96[] calldata topUpAmounts
-  ) external onlyOwner {
+  ) external onlyRoleOrAdminRole(EXECUTOR_ROLE) {
```

Comment:
nitpick, but I think this name is a bit confusing. Can we just call it `onlyAdminOrExecutor`?